lfucg-exactions/client/js/components/LotSummary.js

```diff
@@ -217,13 +221,13 @@ class LotSummary extends React.Component {
                                         </div>
                                     </div>
                                     <div className="col-xs-12">
-                                        <h3 className="col-xs-12 ">Current Exactions: {lots.lot_exactions && lots.lot_exactions.current_exactions}</h3>
-                                        <p className="col-md-8 col-xs-12">Address: {lots.address_full}</p>
-                                        <p className="col-md-4 col-xs-6">Lot Number: {lots.lot_number}</p>
-                                        <p className="col-md-4 col-xs-6 ">Permit ID: {lots.permit_id}</p>
-                                        <p className="col-md-4 col-xs-6">Latitude: {lots.latitude}</p>
-                                        <p className="col-md-4 col-xs-6">Longitude: {lots.longitude}</p>
-                                        <p className="col-md-4 col-xs-6">Approved: {lots.is_approved ? 'Approved' : 'Not Approved'}</p>
+                                        <h3 className="col-xs-12 ">Current Exactions: {currentLot && currentLot.lot_exactions && currentLot.lot_exactions.current_exactions}</h3>
```

Comment:
If you are going to put a currentLot requirement on each one, put it in a higher div level so you only have to check once.